src/lib/Libwin/win_remote_shell.c

```diff
@@ -497,8 +517,12 @@ run_command_si_blocking(STARTUPINFO *psi, char *command, DWORD *p_returncode, in
 			close_valid_handle(&(pi.hProcess));
 			close_valid_handle(&(pi.hThread));
 		}
-		else
```

Comment:
Please add error logging for 
https://github.com/PBSPro/pbspro/blob/bb1fa7be4315449eb94bb63e280b260c4cec3c0f/src/lib/Libwin/win_remote_shell.c#L511
https://github.com/PBSPro/pbspro/blob/bb1fa7be4315449eb94bb63e280b260c4cec3c0f/src/lib/Libwin/win_remote_shell.c#L513
https://github.com/PBSPro/pbspro/blob/bb1fa7be4315449eb94bb63e280b260c4cec3c0f/src/lib/Libwin/win_remote_shell.c#L516